satpy/etc/readers/satpy_cf_nc.yaml

```diff
@@ -17,3 +17,4 @@ file_types:
         file_patterns:
          - '{platform_name}-{sensor}-{resolution_type}-{start_time:%Y%m%d%H%M%S}-{end_time:%Y%m%d%H%M%S}.nc'
          - '{platform_name}-{sensor}-{start_time:%Y%m%d%H%M%S}-{end_time:%Y%m%d%H%M%S}.nc'
+         - '{filename}.nc'
```

Comment:
For consistency with the GRIB2 reader, could this be `{stem}.nc`?  `stem` would also be more accurate, as `.nc` is part of the filename.

Reply:
@gerritholl I would be ok with that. I just added `filename` based on the `generic_image` reader config.

Reply:
fine by me.

Reply:
I vote for `stem` also. The generic_image can be updated later.